qiskit/providers/aer/openpulse/pulse_system_model.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+# pylint: disable=eval-used, exec-used, invalid-name, missing-return-type-doc
+
+"System Model class for system specification for the PulseSimulator"
+
+from collections import OrderedDict
+from .hamiltonian_model import HamiltonianModel
+
+
+class PulseSystemModel():
+    """PulseSystemModel containing all model parameters necessary for simulation.
+    """
+    def __init__(self,
+                 hamiltonian=None,
+                 qubit_freq_est=None,
+                 meas_freq_est=None,
+                 u_channel_lo=None,
+                 qubit_list=None,
+                 channels=None,
+                 dt=None):
+        """Basic constructor.
+        """
+
+        # default type values
+        self.qubit_freq_est = qubit_freq_est
```

Comment:
Not sure we should have public attributes for these. Maybe either store them in a `PulseDefaults` object that is returned by `.defaults` or have setter/getter methods for them

Reply:
Marked to figure out later along with the `digest` stuff.